modules/v2/variables.tf

```
@@ -126,7 +126,7 @@ variable "service_account_project_roles" {
 
 variable "members" {
   type        = list(string)
-  description = "Users/SAs to be given invoker access to the service"
+  description = "Users/SAs to be given invoker access to the service. This can be `allUsers`, `allAuthenticatedUsers` or any other Users/SAs. For a comprehensive list of Users/SAs, please refer to the provided link: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service_iam#member/members-1."
```

Could you update the description like below and regenerate the metadata!

```suggestion
  description = "Users/SAs to be given invoker access to the service. Grant invoker access by specifying the users or service accounts (SAs). Use allUsers for public access, allAuthenticatedUsers for access by logged-in Google users, or provide a list of specific users/SAs. See the complete list of available options: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service_iam#member\/members-1"
```